apps/src/templates/teacherDashboard/EditSectionForm.jsx

```diff
@@ -251,6 +311,44 @@ GradeField.propTypes = {
   validGrades: PropTypes.arrayOf(PropTypes.string).isRequired
 };
 
+const LoginTypeField = ({value, onChange, validLoginTypes, disabled}) => {
```

Comment:
Do you think we should add unit tests for this component? - there is a good amount of tricky logic going on here to determine what we show, and I think we should capture all of it in tests.

Reply:
Done! I didn't  add coverage for the whole functionality of the EditSectionForm but added tests for the `LoginTypeField`.